aws/src/test/java/org/apache/iceberg/aws/TestAwsClientFactories.java

```diff
@@ -56,24 +52,6 @@ public void testLoadCustom() {
         "should load custom class", AwsClientFactories.from(properties) instanceof CustomFactory);
   }
 
-  @Test
```

Comment:
[question] is the coverage for the code-path, covered in some other UT ? If not would recommend adapting this UT to the changes.

Reply:
Thank you for pointing this out. I forgot to add the test after deleting the original one. Since now the `applyS3CredentialConfigurations` locate in `AwsProperties.java`, I added a new test called `testS3FileIoCredentialsConfiguration()` in `TestAwsProperties.java` with similar testing logic like the deleted one.